app/metrics/support/counter_support.rb

```diff
@@ -7,6 +7,10 @@ def fetch_and_format(aggregate = nil)
     # The block on the deep_merge ensures that the final values
     # are added.  Without it, the value is the last value passed in.
     ordered_counters.each_with_object({}) do |values, result|
+      # Rails casts PG numeric as BD, which causes problems in the addition.
```

Comment:
Actually, it wasn't.  It looked like it was throwing error because it failed to convert the BigDecimal from its class representation.  However, the problem was with a failure elsewhere.  I left it in by accident.  Removing now.